reflex/compiler/compiler.py

```diff
@@ -98,11 +98,16 @@ def _compile_contexts(state: Type[State]) -> str:
     Returns:
         The compiled context file.
     """
-    return templates.CONTEXT.render(
-        initial_state=utils.compile_state(state),
-        state_name=state.get_name(),
-        client_storage=utils.compile_client_storage(state),
-        is_dev_mode=os.environ.get("REFLEX_ENV_MODE", "dev") == "dev",
+    is_dev_mode = (os.environ.get("REFLEX_ENV_MODE", "dev") == "dev",)
```

Comment:
```suggestion
    is_dev_mode = os.environ.get("REFLEX_ENV_MODE", "dev") == "dev"
```

is_dev_mode should just be a bool